docs/source/quickstart.rst

```diff
@@ -0,0 +1,182 @@
+   - This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPretraining model).
+   - This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
+   Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-cased and are newly initialized: ['classifier.weight', 'classifier.bias']
+   You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
+   >>> tokenizer = AutoTokenizer.from_pretrained('bert-base-cased')
+
+.. tab:: TensorFlow
+
+   >>> from transformers import TFAutoModelForSequenceClassification, AutoTokenizer
+   >>> model = TFAutoModelForSequenceClassification.from_pretrained("bert-base-cased")
+   Some weights of the model checkpoint at bert-base-cased were not used when initializing TFBertForSequenceClassification: ['nsp___cls', 'mlm___cls']
+   - This IS expected if you are initializing TFBertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPretraining model).
+   - This IS NOT expected if you are initializing TFBertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
+   Some weights of TFBertForSequenceClassification were not initialized from the model checkpoint at bert-base-cased and are newly initialized: ['dropout_37', 'classifier']
+   You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
+   >>> tokenizer = AutoTokenizer.from_pretrained('bert-base-cased')
+
+Tokenize the dataset
+--------------------
+
+The next step is to tokenize the text in order to build sequences of integers the model can understand. Encode the entire dataset with :func:`datasets.Dataset.map`, and truncate and pad the inputs to the maximum length of the model. This ensures the appropriate tensor batches are built.
+
+.. code-block::
+
+   >>> def encode(examples):
+   ...     return tokenizer(examples['sentence1'], examples['sentence2'], truncation=True, padding='max_length')
+   
+   >>> dataset = dataset.map(encode, batched=True)
+   100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.75it/s]
```

Comment:
Not sure if showing the progress bar aids understanding, so remove? (No strong opinion if you want to keep it :))